.gitignore

```diff
@@ -25,3 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
 node_modules
+
+# Jetbrains metafiles
+.idea
```

Comment:
This slipped in change doesn't belong in this PR or commit. Can you please remove.